### DISCUSSION TITLE
Mapping popup view in buffer

### DISCUSSION THREAD

**Discussion by Kayzels** [LazyVim/LazyVim#2913] [Q&A]
I'm assuming Noice is responsible, but I'm not sure.

When I do a `:checkhealth`, WhichKey reports conflicting keymaps for `gc`, `ga`, `ys` and `yS`. So I typed `:map gc` to see the conflict. A notification popup shows up with it, but disappears before I can read it. Viewing the noice history doesn't show the message.

Can I view this popup message in a buffer, so that it doesn't expire, and I can actually read it?

**Comment by andreacfromtheapp** [comment]
usually they are logged to `:messages`, have you tried it already?

**Reply by Kayzels** [reply]
I already did, yes. I tried that, `:NoiceHistory` and `:NoiceLast`, and the results didn't show the details for the mapping.

What I ended up doing was `:map`, which opened a buffer, and then I searched for the conflicting keys. From what I could tell, it was a false positive any way.

**Comment by dpetka2001** [comment]
You can use `NoiceHistory` or `NoiceLast` for only the last message.

**Reply by Kayzels** [reply]
Thanks, but unfortunately those didn't help.

**Reply by dpetka2001** [reply]
Strange, this works for me. I do `verbose map gc` and then `NoiceLast` gives me the output in a floating window by default. Have you made some changes to `noice.nvim`?

**Reply by Kayzels** [reply]
I checked the changes I've made for Noice, but there's nothing that seems like it should be doing that. The only changes I've made are related to the markdown highlights.

Its possible that my Neovim is broken, though. I had to force close it recently, and I think that affected the shada file.
